libsolidity/interface/CompilerStack.cpp

```diff
@@ -70,6 +72,26 @@ void CompilerStack::setRemappings(vector<string> const& _remappings)
 		r.prefix = colon == eq ? string(remapping.begin(), eq) : string(colon + 1, eq);
```

Comment:
I'm hesitant to change any naming/documentation until @chriseth comes back (I think he maybe had a reason for naming it to that), but I think I'm with you that it needs changing. Personally I've always found that naming them `context`:`remote`=`local` makes the most sense for people being introduced to this.